Unable to start a demo page

### Environment Information 
 
- __Package version(s)__: current master 
- __Browser and OS versions__: MacOS 10.14.4, node 8.11.3, npm 6.4.0

### Please provide a reproduction of the bug in a codepen:
 
README.md says that to run a demo page I just need to:

1. `git clone https://github.com/OfficeDev/office-ui-fabric-react.git`
2. `cd office-ui-fabric-react`
3. `npm install`
4. `npm start`

#### Actual behavior:
 
Probably caused by #8844, `npm start` fails with errors similar to the following one:
```
ERROR in ./src/components/pages/ActivityItemPage.tsx
Module not found: Error: Can't resolve '@uifabric/api-docs/lib/pages/office-ui-fabric-react/ActivityItem.page.json' in '/Users/mistastn/devel/office-ui-fabric-react/apps/fabric-website-resources/src/components/pages'
 @ ./src/components/pages/ActivityItemPage.tsx 7:124-209
 @ ./src/AppDefinition.tsx
 @ ./src/index.tsx
 @ multi /Users/mistastn/devel/office-ui-fabric-react/common/temp/node_modules/.registry.npmjs.org/webpack-dev-server/3.1.14/webpack@4.29.5/node_modules/webpack-dev-server/client?http://localhost:4322 ./src/index.tsx
```

I have to run `npm run build` before `npm start` to run the demo page. 

#### Expected behavior:
 
README.md in sync with what I need to do.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? No
 
Requested priority: (Blocking, High, Normal, Low)

--- a/package.json
+++ b/package.json
@@ -10,11 +10,11 @@
   "scripts": {
     "_rushInstall": "node common/scripts/install-run-rush.js install --purge",
     "_rushBuild": "node common/scripts/install-run-rush.js build",
-    "_rushBuildToExperiments": "node common/scripts/install-run-rush.js build --to @uifabric/experiments",
+    "_rushBuildToFabricResources": "node common/scripts/install-run-rush.js build --to @uifabric/fabric-website-resources",
     "_rushRebuild": "node common/scripts/install-run-rush.js rebuild --production --verbose --lint",
     "_rushRebuildCi": "node common/scripts/install-run-rush.js rebuild --verbose --lint",
     "_rushRebuildFast": "node common/scripts/install-run-rush.js rebuild",
-    "postinstall": "npm run _rushInstall && npm run _rushBuildToExperiments -- --npm-install-mode",
+    "postinstall": "npm run _rushInstall && npm run _rushBuildToFabricResources -- --npm-install-mode",
     "test": "npm run _rushBuild",
     "vrtest": "cd apps && cd vr-tests && npm run screener",
     "a11ytest": "cd apps && cd a11y-tests && npm run test",

